src/GraphQL/Execution/Nodes/ObjectExecutionNode.cs

```diff
@@ -86,6 +86,11 @@ public override bool PropagateNull()
 
         IEnumerable<ExecutionNode> IParentExecutionNode.GetChildNodes() => SubFields ?? Enumerable.Empty<ExecutionNode>();
 
+        /// <summary>
+        /// Returns the selected fields.
+        /// </summary>
+        public virtual SelectionSet SelectionSet => Field?.SelectionSet;
```

Comment:
Adding this member simplified `ExecutionStrategy.SetSubFieldNodes`.  No additional memory requirements here, but one more field in `RootExecutionNode`.

Reply:
OK